api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -558,6 +521,33 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
             .workspace(workspaceMapper.toApiWorkspace(savedWorkspace, toFcWorkspace)));
   }
 
+  // A retry period is needed because the permission to copy files into the cloned workspace is not granted transactionally
```

Comment:
Optional: could consider just leaving this functionality out in favor of https://precisionmedicineinitiative.atlassian.net/browse/RW-3187 being implemented soon (hopefully).

Reply:
I think I'd prefer to keep it since it is causing some failures already in our production. We can remove it once `RW-3187` lands

Reply:
Maybe add a TODO to remind us to do that

Reply:
Added to the ticket